product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,120 @@
 title: Terminology
 ---
 
-The terminology that follows is important for understanding EDB Postgres Distributed functionality and the requirements that it addresses in the realms of high availability, replication, and clustering.
+There are many terms you will come across in EDB Postgres Distributed that you may be unfamiliar with. This page is a list of many of those terms with quick definitions. 
 
-#### Asynchronous replication 
 
-Copies data to cluster members after the transaction completes on the origin node. Asynchronous replication can provide higher performance and lower latency than synchronous replication. However, it introduces the potential for conflicts because of multiple concurrent changes. You must manage any conflicts that arise.
+#### Asynchronous replication
 
-#### Availability
+Copies data to cluster members after the transaction completes on the origin node. Asynchronous replication can provide higher performance and lower latency than synchronous replication. However, asynchronous replication can see a lag in how long changes take to appear in the various cluster members. While the cluster will be [eventually consistent](#eventual-consistency), there will be potential for nodes to be apparently out of sync with each other.
```

Comment:
Thinking rearranging the sentences might make this easier on the brand new reader i.e. 

Asynchronous replication can provide higher performance and lower latency than synchronous replication. With asynchronous replication, the data is copied to cluster members after the transaction completes on the original node. However, asynchronous replication can see a lag in how long changes take to appear in the various cluster members. While the cluster will be [eventually consistent](#eventual-consistency), there will be potential for nodes to be apparently out of sync with each other.

Reply:
That starts by not explaining what it is and then making statements about features and comparing it with something the reader has yet to meet.

```suggestion
A type of replication that copies data to other PGD cluster members after the transaction completes on the origin node. Asynchronous replication can provide higher performance and lower latency than [synchronous replication](#synchrous-replication). However, asynchronous replication can see a lag in how long changes take to appear in the various cluster members. While the cluster will be [eventually consistent](#eventual-consistency), there will be potential for nodes to be apparently out of sync with each other.
```